client/utils/evaluateExpression.ts

```diff
@@ -1,11 +1,18 @@
-function __makeEvaluateExpression(evalInClosure) {
-  return (expr) =>
+type EvalResult = {
+  result: any,
```

Comment:
i haven't looked closely enough at where this is used, but is there any way to avoid using `any` here as well? can we do `unknown` and then cast it later after doing some type guard checks? 

https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-guards-and-differentiating-types